magento/pwa-studio
# Add input section to styleguide

## Description
Add an input section to the styleguide. The prose content may be lorem ipsum, but the values should reflect the the official Venia styles. There are four basic inputs included in this scope:

- checkbox
- radio
- select
- text

## Related Issue
PWA-436.

## Acceptance 
- @jcalcaben should affirm document structure
- @soumya-ashok or @schensley should affirm Venia typographic styles

### Verification Stakeholders

### Specification

### Verification Steps
1. `yarn workspace @magento/venia-styleguide run start`
2. Navigate to each of the four new pages from the left nav
3. Verify the normal and focus states match the new design intent

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
